.github/workflow/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
```

Comment:
- [ ] Great job setting up the `linters.yml` file 👍 . Kindly make sure you also set up  `lighthouse` `stylelint` and `webhint` by following this [article](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse 🙏 . Using linters as a developer would help you learn more about writing DRY code

Reply:
Hi @Ugboaja-Uchechi  
I hope 
Thank you for the review.

I added the stylelint and Webhint also provided a description for the pull request.